web/modules/custom/demo_content/content/rdf_entity/asset_distribution/blue.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
- I don't see the added value that changing to JSON representation brings?
- I wouldn't put these fixtures in the modules themselves. Keeping everything centralized will assure an easy cleanup.
(This module should be removed from the repo befor go live imho)